packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js

```diff
@@ -1474,6 +1474,102 @@ module.exports = function (config, DB) {
       })
     })
 
+    describe('db.securityEventsByUid', () => {
+      let session1, session2, session3, anotherAccountSession, uid, anotherUid
+      const evA = 'account.login', evB = 'account.create', evC = 'account.reset'
+      const addr = '127.0.0.1'
+
+      function insert(uid, addr, name, session) {
+        return db.createSecurityEvent({
+          uid: uid,
+          ipAddr: addr,
+          name: name,
+          tokenId: session
+        })
+      }
+
+      beforeEach( () => {
+        const anotherAccountData = createAccount()
+        db.createAccount(anotherAccountData.uid, anotherAccountData)
```

Comment:
`db.createAccount` returns a promise, to avoid race conditions do something like https://github.com/mozilla/fxa/blob/ebd32d7eccd804d8a9ef359f04b15ac9509811f1/packages/fxa-auth-db-mysql/db-server/test/backend/db_tests.js#L1816.